### ISSUE TITLE
After build nothing is displayed on screen.

### ISSUE THREAD

**Issue by f4004** [edvin/tornadofx#1316]
I fixed all of problems about gradle and building. But I still have a problem. Having clicked **_Run_** I get **_BUILD SUCCESSFUL_** but nothing appear. And program is ended by doing nothing. Why?

Build output:
```
BUILD SUCCESSFUL in 920ms
1 actionable task: 1 executed
Task execution finished.
```

**Comment by SKeeneCode** [comment]
Build what? Are you building the tornadofx library or your own project?

**Comment by f4004** [comment]
I'm build my own project. I'm trying to use TornadoFX library in my project. I've tried[ this code](https://github.com/edvin/tornadofx#what-does-it-look-like-code-snippets).

**Comment by SKeeneCode** [comment]
Put a launch function outside:

**Main.kt**
```
class HelloWorld : View() {
    override val root = hbox {
        label("Hello world")
    }
}

class HelloWorldApp : App(HelloWorld::class, Styles::class)

class Styles : Stylesheet() {
    init {
        label {
            fontSize = 20.px
            fontWeight = FontWeight.BOLD
            backgroundColor += c("#cecece")
        }
    }
}

fun main() {
    launch<HelloWorldApp>()
}
```

and then in your gradle.build.kts:

```
application {
    mainClass.set("MainKt")
}
```

Then run using gradle. Does this work?

**Comment by f4004** [comment]
Thanks for you answer. But it doesn't work anyway. Still same.

**Comment by SKeeneCode** [comment]
Are you sure you're running the `run` task? Gradle > Tasks > Application > run?

![image](https://user-images.githubusercontent.com/32420836/109518921-a36c3380-7aa2-11eb-9d18-100705f16fe6.png)

**Comment by f4004** [comment]
I got new error this time when I tried your code:
> src\main\kotlin\main.kt: (1, 20): Unresolved reference: View

When I tried the code I've wrote, I got this error:

> src\main\kotlin\main.kt: (4, 27): Unresolved reference: HelloWorld

From Gradle > Run section.

**Comment by SKeeneCode** [comment]
Have you got tornadofx imported using `import tornadofx.*`?

**Comment by f4004** [comment]
Yeah now it is working. Thank you. It needs to `import javafx.scene.text.FontWeight`. Thank you again. Can I some question regarding this codes.

**Comment by SKeeneCode** [comment]
No problem. Feel free to ask anything else - if I can answer I will :)

**Comment by f4004** [comment]
- What is the difference between Run from Intellij and Run from Gradle and why nothing is displayed when I clicked Run(from Intellij)?

- Why we used` mainClass ("MainKt")` instead of `mainClassName = "MainKt"`? Actually when I've tried both are works fine.

**Comment by SKeeneCode** [comment]
> * What is the difference between Run from Intellij and Run from Gradle and why nothing is displayed when I clicked Run(from Intellij)?

Are you sure you're actually running the application in intellij? By clicking the green arrow and not just the build button? 
![image](https://user-images.githubusercontent.com/32420836/109551483-88141f00-7ac8-11eb-8cd0-e4301cab6017.png)
I'm guessing there is something mistaken with your set up.

I think I might need to look at your project to determine exactly what is going wrong.

You can read more about exactly what a gradle run task does [here.](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.JavaExec.html)

> * Why we used` mainClass ("MainKt")` instead of `mainClassName = "MainKt"` in build.gradle.kts? Actually when I've tried both are works fine.

`mainClassName = "MainKt"` is deprecated, the other way seems to be the [appropriate way of doing it now](https://stackoverflow.com/questions/64749528/setter-for-mainclassname-string-is-deprecated-deprecated-in-java).

**Comment by f4004** [comment]
> Are you sure you're actually running the application in intellij? By clicking the green arrow and not just the build button?
> ![image](https://user-images.githubusercontent.com/32420836/109551483-88141f00-7ac8-11eb-8cd0-e4301cab6017.png)

As your seen following image, I was clicking green arrow nothing appears after build. Actually when I click Run from top menu appears two section there. kotlinandtornadofx (with Gradle icon) and MainKt (with Kotlin icon) file. When I clicked MainKt, it gives a lot of debug error, but kotlinandtornadofx file doesn't give error but doesn't run anything neither. 
![image](https://user-images.githubusercontent.com/52128581/109554000-bf90c480-7ae4-11eb-8dd6-57eb4adcc936.png)
After first Run in Gradle, kotlinandtornadofx - run section has been added.

**Comment by SKeeneCode** [comment]
>When I clicked MainKt, it gives a lot of debug error

Can you post the errors?

**Comment by f4004** [comment]
I'm sorry to keep you waiting. I've deleted that configuration but I've created a new project and I tried. MainKt works fine. But why my previous project had Gradle symbol? Seem like MainKt and kotlinandtornadofx is run by another compiler or something.


> ![image](https://user-images.githubusercontent.com/52128581/109554000-bf90c480-7ae4-11eb-8dd6-57eb4adcc936.png)

**Comment by SchweinchenFuntik** [comment]
well, without the configuration window, it is difficult to understand which task you are launching, if you yourself did not specify the name, then perhaps you have a custom task

**Comment by f4004** [comment]
> well, without the configuration window, it is difficult to understand which task you are launching

In fact, I wonder, there is only one task to run: it is project. Perhaps I don't know tasks.

**Comment by SchweinchenFuntik** [comment]
terminal (linux): `./gradlew run` (wrapper) or `gradle run`
